Documentation/flexMigration/flex-to-csi-migration.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Migration
+weight: 4100
+indent: true
+---
+
+# Flex to CSI Migration
+
+The tool [persistent-volume-migrator](https://github.com/ceph/persistent-volume-migrator) aims to help migrate Flex rbd Volumes to Ceph-Csi.
+
+## Prerquisite
+
+This guide assumes you have created a Rook cluster with Flex
+1) Minimum ceph version should be Ceph-CSI supported.
+2) Volume should be in BOUND state and should **not** be in use.
+3) CSI storageClass should be created in which you want to migrate.
+   1) Example: CSI [storageclass](https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/csi/rbd/storageclass.yaml).
+
+**NOTE**: The Migration procedure will come with a downtime; As we need to scale down the applications using the Volumes before migration.
+
+## Steps
+1) Create rbac required to run migrator pod
+   1) `kubectl create -f cluster/examples/kubernetes/ceph/flex-migration/rbac.yaml`
+2) Create migrator pod
+   1) `kubectl create -f cluster/examples/kubernetes/ceph/flex-migration/migrator.yaml`
```

Comment:
Link directly to the other repo to create the pod. 
```suggestion
   1) `kubectl create -f https://github.com/ceph/persistent-volume-migrator/blob/main/tests/manifests/migrator.yaml
```

Reply:
will update soon

Reply:
done